core/Category/Subcategory.php

```diff
@@ -143,4 +150,22 @@ public function getOrder()
     {
         return $this->order;
     }
+
+    /**
+     * Get the help text (if any) for this category.
+     * @return null
+     */
+    public function getHelp()
+    {
+        return $this->help;
+    }
+
+    /**
+     * Sets the help text for this category.
+     * @param null $help
+     */
+    public function setHelp($help)
```

Comment:
Might it make sense to add the `setHelp` to the `Category` class as well - for consistency reasons?

Reply:
It should be in both

Reply:
Actually I had it in both at first, but the help string can be long so instead I just forced category/subcategory to override getHelp(). I forgot to change this file at the time.